src/pages/index.js

```diff
@@ -40,6 +40,24 @@ import workforceDevelopmentIcon from "../files/icons/workforce-dev-icon.svg";
 import arrowIcon from "../files/icons/arrow-icon.svg";
 
 const HomePage = ({ data }) => {
+  useEffect(() => {
+    const faders = document.querySelectorAll(".fade-in");
```

Comment:
Need to figure out if this should be done with useRefs instead.

Reply:
@Yodart here's some feedback from the frontend-channel about possible ways to add animations to React:

---

I would look and see if the React CSSTransition group has anything you can use as a starting place https://reactjs.org/docs/animation.html

daniel.gading  3 hours ago
If that doesn't work, as long as you are using functional components ,you can use the useRef hook to access DOM elements and probably make the updates needed https://reactjs.org/docs/hooks-reference.html#useref


EricH:spiral_calendar_pad:  2 hours ago
The few times I've tried transitions like that I made use of the CSSTransition: http://reactcommunity.org/react-transition-group/css-transition. Worked great, and was pretty straightforward to use, but I think that would only handle the fade-in part. Not sure how you'd handle trigging it when scrolled into view?

EricH:spiral_calendar_pad:  2 hours ago
This may be helpful too? https://www.gatsbyjs.com/plugins/gatsby-plugin-scroll-reveal/

Reply:
If it takes too long ti implement, it's ok to spin the refactoring of the animations off into a new ticket.

Reply:
@iris-i I'll look into using `useRef` to access the components however using those animations library strike me as overkill for just a simple fade-in animation on-scroll, a pattern that I expect to be maintained thruout the application, I don't see us doing complex animations, would still advice we make that change?

Reply:
@Yodart I'd recommend using useRefs at the very least. I agree, we can skip the animation libraries if we don't anticipate adding complex animations to the site.